operations/helm/tests/test-enterprise-configmap-values-generated/mimir-distributed/templates/gateway/gateway-svc.yaml

```diff
@@ -15,10 +15,14 @@ metadata:
 spec:
   type: ClusterIP
   ports:
-    - port: 8080
+    - port: 80
       protocol: TCP
       name: http-metrics
       targetPort: http-metrics
+    - port: 8080
+      protocol: TCP
+      name: legacy-http-metrics
+      targetPort: http-metrics
```

Comment:
Do you know how this will affect the service monitors - will the gateway be scraped twice?

Reply:
I tested it and it's not scraped twice. It only discovers the port named `http-metrics`, the new `legacy-http-metrics` does not match.

Reply:
👍 https://github.com/grafana/mimir/blob/9914a64126abc58b7368b1cfb08f5cabd63968c8/operations/helm/charts/mimir-distributed/templates/lib/service-monitor.tpl#L42